setup.cfg

```diff
@@ -63,9 +63,40 @@ ignore =
 formats = sdist.tgz,bdist_wheel
 
 [mypy]
-ignore_missing_imports = True
 no_implicit_optional = True
 strict_equality = True
 warn_redundant_casts = True
 warn_return_any = True
 warn_unused_configs = True
+
+# These packages don't have type annotations yet.
+[mypy-xml.dom.*]
+ignore_missing_imports = True
+[mypy-parser.*]
+ignore_missing_imports = True
+[mypy-bdb.*]
+ignore_missing_imports = True
+[mypy-importlib_metadata.*]
+ignore_missing_imports = True
+[mypy-zope.interface.*]
+ignore_missing_imports = True
+[mypy-twisted.*]
+ignore_missing_imports = True
+[mypy-numpy.*]
+ignore_missing_imports = True
+[mypy-py.*]
+ignore_missing_imports = True
+[mypy-wcwidth.*]
+ignore_missing_imports = True
+[mypy-pluggy.*]
+ignore_missing_imports = True
+[mypy-more_itertools.*]
+ignore_missing_imports = True
+[mypy-packaging.*]
+ignore_missing_imports = True
+[mypy-argcomplete.*]
+ignore_missing_imports = True
+[mypy-xdist.*]
+ignore_missing_imports = True
+[mypy-colorama.*]
+ignore_missing_imports = True
```

Comment:
> This will catch typos in imports and misconfigurations and such.

Can you elaborate, please?
Do you mean that it will error out when using an import without type definitions that is not listed here?
Or something more/else?

I find it a bit bad having to manage a list of used 3rd party packages here after all.

Reply:
> Do you mean that it will error out when using an import without type definitions that is not listed here?

Indeed - the other option would be to ignore missing imports globally, but IMHO this is better. Hopefully, over time, the list will get shorter as there are annotations for more packages.

Reply:
Yes, as @The-Compiler said, enabling this option globally means any error in imports will go unnoticed by the static type checker, and I think having such a check when developing is really nice.

The other alternative is to add `# type: ignore` on each import but that seems worse.

The mypy docs have some discussion on this: https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports

Reply:
I'd prefer we just always have `ignore_missing_imports = True` -- the trade-off here isn't worth the noise / tedium

Reply:
I agree with @asottile here - typos in imports are not really an argument, they would be caught otherwise.

Reply:
I've removed this commit.